care/abdm/api/viewsets/auth.py

```diff
@@ -346,10 +356,14 @@ def post(self, request, *args, **kwargs):
                 }
             )
         except Exception as e:
+            logger.warning(
+                f"Error: RequestDataView::post failed to notify (health-information/notify). Reason: {e}",
+                exc_info=True,
+            )
             return Response(
                 {
                     "detail": "Failed to notify (health-information/notify)",
-                    "error": str(e),
+                    "error": "Error: Failed to notify",
```

Comment:
```suggestion
```
no need for this key as we only use the detail to show the error message 
https://github.com/coronasafe/care_fe/blob/51b02c26666877f2b0b481f0476a9b34acba6743/src/Redux/fireRequest.tsx#L158